gnuboy-go/components/gnuboy/palettes.h

```diff
@@ -5,14 +5,15 @@
 // Our custom colorization palettes
 static const uint16_t dmg_palettes[][4] = {
 	{ 0x6BDD, 0x3ED4, 0x1D86, 0x0860 }, // GB_DEFAULT_PALETTE
-	{ 0x67FE, 0x3F12, 0x29A0, 0x1060 }, // GB_G&W_PALETTE
+	{ 0x6BFC, 0x3B12, 0x31C8, 0x1060 }, // GB_G&W_PALETTE
```

Comment:
Which palette is accurate here and where do the values come from?

Reply:
The colors were too warm, I realized while playing that they weren't exactly the same.  so I readjusted them, they are more "greenish"